Makefile

```diff
@@ -44,37 +35,31 @@ format-solidity:
 format: format-go format-shell format-solidity
 
 .PHONY: test
-test: init
+test: 
 	./scripts/run-unit-tests.sh 2>&1 | tee test.log
 
 .PHONY: test-integration
-test-integration: init
+test-integration: 
 	./scripts/run-integration-tests.sh 2>&1 | tee test-integration.log
 
 # Install all the binaries into $HOME/go/bin (or alternative GOPATH bin directory)
 .PHONY: install
-install: init
+install: 
 	go install ./cmd/...
 
 # Install swapd and swapcli into this directory (top of the project)
 .PHONY: build
-build: init
+build: 
 	./scripts/build.sh
 
-# WARNING: this should not be used in production, as the DLEq prover has been stubbed out and now proves nothing.
-.PHONY: build-go
-build-go:
-	GOBIN="$(CURDIR)/bin" go build -tags=fakedleq ./cmd/swapd
-	GOBIN="$(CURDIR)/bin" go build -tags=fakedleq ./cmd/swapcli
-
-# WARNING: this should not be used in production, as the DLEq prover has been stubbed out and now proves nothing.
-.PHONY: build-go-darwin
-build-go-darwin:
-	GOOS=darwin GOARCH=arm64 $(MAKE) build-go
+.PHONY: build-darwin
+build-darwin:
```

Comment:
Maybe we should add a comment so that no one uses this target when compiling on macos?  Something like:
```
# Test macos/arm build from linux. Use "make build" if compiling on macos.
```
If we want to provide release binaries at some point for linux/arm, linux/intel, macos/arm, macos/intel ..., I can script something up that will replace this target.

Reply:
good point, added! and yep, that would be nice to have soon